JumpscaleCore/core/BASECLASSES/JSConfigsBCDB.py

```diff
@@ -83,19 +83,21 @@ def _new(self, name, jsxobject=None, save=True, **kwargs):
                 jsxobject = self._model.new()
             jsxobject.name = name
 
-        jsxobject._autosave = True
```

Comment:
Having autosave set to True by default ends up saving the object whenever an attribute is changed and it doesn't respect that it might have been created with save=False